views/blog.py

```diff
@@ -1,8 +1,10 @@
 import requests
 import json
 
+from django.http import HttpResponse
```

Comment:
Why has this been imported?

Reply:
i was trying sending message with diffferent types of responses. will remove that